examples/gno.land/p/demo/grc/grc721/igrc721_royalty.gno

```diff
@@ -0,0 +1,18 @@
+package grc721
+
+import (
+	"std"
+)
+
+// IGRC2981 follows the Ethereum standard 
+type IGRC2981 interface {	
+	// RoyaltyInfo retrieves royalty information for a tokenID and salePrice.
+	// It returns the payment address, royalty amount, and an error if any.
+	RoyaltyInfo(tokenID TokenID, salePrice uint64) (std.Address, uint64, error)
```

Comment:
I think it makes sense for this function to simply return `RoyaltyInfo` - this way would follow Go convention more.

Reply:
I disagree, unless we change RoylatyInfo to match the current return signature

this method signature is the same as erc2981

Reply:
I'm of the opinion that we should not follow spec syntax-wise, but functionality-wise, especially since we are using a completely different language to implement this. As far as I see, the `RoyaltyInfo` type contains exactly `std.Address` & `uint64`, so I don't see a problem in just using it instead of multiple return variables.

Reply:
RoyaltyInfo contains
```go
Percentage     uint64      // Percentage is the royalty percentage.
```
where the second return in erc2981 is an amount in atomics

Reply:
this confusion also shows why you are right on the "return an object" part :)

Reply:
@n0izn0iz I agree, @leohhhn the field uint64 in struct RoyaltyInfo stands for percentage, but the second return param is royaltyAmount (= percentage * salePrice), unless we should define new reponse struct for more clarity if change